tests/server/mocks/expected-data.js

```diff
@@ -15,6 +15,24 @@ module.exports = {
       stage: 'login_success'
     }
   ],
+  hooks: [
+    {
+      script: 'hook1 script',
+      name: 'Hook-1',
+      triggerId: 'client-credentials',
+      enabled: true,
+      secrets: {},
```

Comment:
Is there a reason `secrets` and `dependecies` are always dropped?
Would this lead to the secrets and dependencies being removed?

Reply:
Looking at the test, this is the expected data based on the data we pass in: https://github.com/auth0-extensions/auth0-deploy-extensions/pull/44/files#diff-298ad2b70dc85b087647efb12f2e14b8R85

The sourcecontrol-extension-tools library is what will handle the diffing and determine if  it needs to be updated, removed, etc.

Reply:
Awesome, thanks for the clarifications!